scripts/prs-since-last-tag

```diff
@@ -11,8 +11,8 @@ version=$(jq -r .version package.json)
 
 token=$(git config --global github.token)
 
-echo "# $version ($(date +%d.%m.%Y))"
-for pr in $(git log --grep "Merge pull request" "$last_tag"..HEAD | sed -nEe 's/.*#([0-9]+).*/\1/p'); do
+echo "# $version ($(date +%F))"
```

Comment:
![grafik](https://user-images.githubusercontent.com/1606004/49933693-9d430a00-fecc-11e8-80b9-8dafbb4646dd.png)

Reply:
It's not even what most americans use ;)

Reply:
😅 yes, true.

TBH I personally switched to `+%F` a long time ago when naming files since it's a no brainer to sort files historically that way.

Reply:
Yup! That's one of the big reasons I think iso8601 is the way to go!